### PR TITLE
[User roles] Allow members to view transfer modal

### PR DESCRIPTION
**Pull request by malted**
Also, adds this infobox
image

### CODE REVIEW

File: app/views/events/_transfer_form.html.erb [hunk 1]
```diff
@@ -13,11 +13,13 @@
         <strong>Mailed check</strong>
       <% end %>
 
-      <%= link_to (paypal_transfers_airtable_form_url(embed: false, event: @event, user: @current_user) unless @event.demo_mode?), class: "text-decoration-none btn--form-option #{"btn--form-option--disabled" if @event.demo_mode?}", data: { turbo_frame: "_top" } do %>
+      <%= link_to (paypal_transfers_airtable_form_url(embed: false, event: @event, user: @current_user) unless @event.demo_mode? || !policy(@event).new_transfer?), class: "text-decoration-none btn--form-option #{"btn--form-option--disabled" if @event.demo_mode? || !policy(@event).new_transfer?}", data: { turbo_frame: "_top" } do %>
```
Comment (garyhtou) [inline app/views/events/_transfer_form.html.erb]: I wonder if we want to move the demo_mode? check into new_transfer?

File: app/views/events/transfers.html.erb [hunk 2]
```diff
@@ -147,6 +147,25 @@
...
+    <% if !policy(@event).new_transfer? %>
+      <div class="card border b--info pb0 mt3 mb3">
+        <div class="flex items-center info">
+          <%= inline_icon "info", size: 32, class: "mr1" %>
+          <p class="bold mt0 mb0">You can't transfer money.</p>
+        </div>
+        <ul>
+          <li>
+            This is because you're not a manager.
+            (You're a <%= OrganizerPosition.find_by(user: current_user, event: @event).role %>.)
```
Comment (garyhtou) [inline app/views/events/transfers.html.erb]:
```suggestion
            (You're a <%= OrganizerPosition.find_by(user: current_user, event: @event).role %>)
```

File: app/views/events/_transfer_form.html.erb (hunk 1 above)
Comment (malted) [reply app/views/events/_transfer_form.html.erb]: Done!

**Review by garyhtou — APPROVED** [review]
Good to now, the forms themselves need the UI disabled. but this is good to merge

**Pull request merged**